packages/external-integration/src/lib/external-payment-strategy.ts

```diff
@@ -64,6 +65,12 @@ export default class ExternalPaymentStrategy implements PaymentStrategy   {
         return Promise.resolve();
     }
 
+    protected redirectUrl(url: string): Promise<void> {
+        return new Promise(() => {
+            this._formPoster.postForm(url, { });
```

Comment:
No need to create an empty object.

Reply:
data isn't an optional parameter.
![image](https://user-images.githubusercontent.com/4907027/204634107-f235ffef-bd0c-436e-86ec-aa8aa0a79ef4.png)